data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -923,3 +923,8 @@ popup-separator-menu-item {
   }
 }
 
+.metered-data-switch { width: 48px; }
+.metered-data-switch {
```

Comment:
This is SASS, an so we should take advantage of its capabilities to define things hierarchical: please only declare one `.metered-data-switch` block in this file (i.e. move the `width` statement in the first block to the second one)